playground/frontend/playground_components/lib/src/widgets/run_or_cancel_button.dart

```diff
@@ -49,7 +50,10 @@ class RunOrCancelButton extends StatelessWidget {
               (_) => PlaygroundComponents.toastNotifier.add(_getErrorToast()),
             );
       },
-      runCode: () {
+      runCode: () async {
+        await AnalyticsService.get().trackRunExample(
+          playgroundController.selectedExample!.name,
+        );
```

Comment:
Maybe not await most of the events? Except maybe feedback because the event is the business logic there.

Reply:
Should the event calls be wrapped into unawaited or only `_safeSendEvent`?